lib/devtools/rake/mutant.rb

```diff
@@ -0,0 +1,63 @@
+module Devtools
+  module Rake
+    # Mutant metric runner
+    class Mutant
+      include Procto.call(:verify),
+              Adamantium,
+              Anima.new(
```

Comment:
Why dont you simply pass the `Devtools::Config::Mutant` object into the `Devtools::Rake::Mutant` one? Reducing the need to duplicate the attribute definition, both here and in any call side?